import stdio;
import mat.exception;

int main() {
    return 1 + 2 + 2 - 1 * 3;
}
